backend/entityservice/views/run/description.py

```diff
@@ -2,28 +2,46 @@
 from structlog import get_logger
 
 from entityservice import database as db
+from entityservice.database import delete_run_data, get_similarity_file_for_run
 from entityservice.views.auth_checks import abort_if_run_doesnt_exist, abort_if_invalid_results_token
 from entityservice.views.serialization import RunDescription
+from entityservice.tasks import delete_minio_objects
 
 logger = get_logger()
 
 
 def get(project_id, run_id):
     log = logger.bind(pid=project_id, rid=run_id)
-    logger.info("request description of a run")
-    # Check the project and run resources exist
-    abort_if_run_doesnt_exist(project_id, run_id)
+    log.info("request description of a run")
+    authorize_run_detail(project_id, run_id)
+    log.debug("request for run description authorized")
```

Comment:
Should this log  be in the method `authorize_run_detail` instead of here?

Reply:
It could - I kept it separate because the `authorize_run_detail` can authorize either deletion or getting the run description.